Packs/NetQuestOMX/Integrations/NetQuestOMX/NetQuestOMX.py

```diff
@@ -0,0 +1,521 @@
+        raise DemistoException(
+            f"An error was occurred when creating an IP list with {name=}."
+            f" This may indicate a list with this name already exists."
+        ) from e
+
+    return CommandResults(readable_output=f"Successfully created a new instance of {name}")
+
+
+def address_list_rename_command(client: Client, args: dict) -> CommandResults:
+    """
+    This function only meant to change the name of the list.
+    Nothing else. If we try to give as a new_name, an existing list name, it will fail, and we’ll get an error.
+
+    Returns:
+        A CommandResults containing a success indication or a DemistoException.
+    """
+    new_name = args["new_name"]  # a required argument - The new name for an existing address list
+    existing_name = args["existing_name"]  # a required argument - Name of the existing address list that we want to modify
+
+    try:
+        client.address_list_rename_request(new_name, existing_name)
+    except Exception as e:
+        raise DemistoException(
+            f"An error was occurred when renaming {existing_name} IPs list to {new_name}."
+            f" Make sure {existing_name} exists and that {new_name} doesn't."
+        ) from e
+
+    return CommandResults(readable_output=f"Successfully renamed {existing_name} to {new_name}")
+
+
+def address_list_delete_command(client: Client, args: dict) -> CommandResults:
+    """
+    This function deletes the list by the given list's name.
+
+    Returns:
+        A CommandResults containing a success indication or a DemistoException.
+    """
+    list_name_to_delete = args["name"]  # a required argument - The name of the address list to delete
+
+    try:
+        client.address_list_delete_request(list_name_to_delete)
+    except Exception as e:
+        raise DemistoException(
+            f"An error was occurred when deleting the {list_name_to_delete} IPs list."
+            f" Make sure {list_name_to_delete} exists."
+        ) from e
+
+    return CommandResults(readable_output=f"Successfully deleted {list_name_to_delete} list")
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    return 'ok'
+
+
+def add_time_to_events(events: list[dict]):
+    """
+    Adds the _time key to the events.
+    Args:
+        events: list[dict] - list of events to add the _time key to.
+    Returns:
+        list: The events with the _time key.
+    """
+    for event in events:
+        create_time = datetime.utcnow()  # The events do not contain a timestamp
+        event["_time"] = create_time.strftime(DATE_FORMAT)  # type: ignore[union-attr]
```

Comment:
```suggestion
    create_time = datetime.utcnow().strftime(DATE_FORMAT)
    for event in events:
        event["_time"] = create_time  # type: ignore[union-attr]
```
ensures all events have the same timestamp, and saves calculating `utcnow()` over and over for each event